templates/components-custom/ec-component-card/card-thumbnail.html.twig

```diff
@@ -0,0 +1,150 @@
+{#
```

Comment:
The problem here is that we are not really adding functionality to the card component, we are simply reusing parts of it. The INNO team already stablished that they are not going to use the list item component for ECL2 so the implementation we do will be entirely up to us. That being said, we could simply not call this template card-thumbnail to avoid confusion and call it navigation-thumbnail and host it in the list-item folder or something like that.

Reply:
Yeah we shouldn't use patching to add entire components, in this case we can maintain them on our side. We could maybe check if ECL Twig wants to maintain them instead, but for now let's proceed with by creating a new template.